scripts/taxonomy.js

```diff
@@ -119,6 +119,10 @@ export async function getTaxonomy() {
 
         getCategory: function (cat) {
           return this.node.querySelector(`[data-type="${cat}"]`);
+        },
+
+        getCategoryTitle: function (cat) {
+          return this.node.querySelector(`[data-type="${cat}"] > p`).innerText;
```

Comment:
Missing a `trim` to be sure.

Reply:
Where is the method used ?

Reply:
Here: https://github.com/adobe/theblog/blob/98d27588ecdd9e610c2bda1929e357957505ee87/scripts/filters.js#L210 (missed a file? 😉)

Reply:
Weird, too many changes in filters.js, it was collapsed!

Reply:
Yeah, I noticed that, too. Not helfpul 😄